content/en/data_jobs/databricks.md

```diff
@@ -71,6 +71,16 @@ This setup does not work on Databricks clusters in <strong>Standard</strong> (fo
 1. Click **Save Databricks Workspace** at the bottom of the browser window.
    {{< img src="data_jobs/databricks/configure-data-jobs-monitoring-existing.png" alt="In the Datadog-Databricks integration tile, Datadog Agent Setup for a Databricks workspace already added to the integration. Datadog can install and manage a global init script." style="width:100%;" >}}
 
+Optionally, you can add custom host tags to your Databricks clusters by configuring the following environment variable in the Advanced Configuration section of your cluster in the Databricks UI or as [Spark env vars][2] with the Databricks API:
```

Comment:
```suggestion
Optionally, you can add tags to your Databricks cluster and Spark performance metrics by configuring the following environment variable in the Advanced Configuration section of your cluster in the Databricks UI or as [Spark env vars][2] with the Databricks API:
```

Reply:
@aboitreaud what API are you referring to at the end: "with the Databricks API" ?